start_geocint.sh

```diff
@@ -2,23 +2,27 @@
 
 # it is the script, that runs the pipeline: it checks required packages, cleans and runs targets, posts info messages
 # Set variables from the configuration file
-. ~/config.inc.sh
+echo ${GEOCINT_WORK_DIRECTORY}
+. ${GEOCINT_WORK_DIRECTORY}/config.inc.sh
 export PATH_ARRAY GENERAL_FOLDER OPTIONAL_DIRECTORIES PRIVATE_REPO_NAME
 export UPDATE_RUNNER UPDATE_OSM_LOGIC UPDATE_PRIVATE RUN_TARGETS
 export SLACK_CHANNEL SLACK_BOT_NAME SLACK_BOT_EMOJI SLACK_KEY USER_NAME
+export CHECK_INSTALLATIONS_BEFORE_RUN
 
 # initialize the pipeline interrupt function for the case when the pipeline can't touch make.lock
 cleanup() {
-  rm -f ~/$GENERAL_FOLDER/make.lock
+  rm -f ${GEOCINT_WORK_DIRECTORY}/$GENERAL_FOLDER/make.lock
```

Comment:
```suggestion
  rm -f ${GEOCINT_WORK_DIRECTORY}/${GENERAL_FOLDER}/make.lock
```

here and below in this file

Reply:
we don't need {} for variables, that were exported inside of bash, but should use them for environment variables